compiler/il/OMRILOps.hpp

```diff
@@ -261,7 +261,9 @@ class ILOpCode
    bool isTwoChildrenAddress()  const { return isArrayRef(); }
    bool isCase()                const { return getOpCodeValue() == TR::Case; }
    bool isAnchor()              const { return getOpCodeValue() == TR::compressedRefs; }
-   bool isOverflowCHK()         const { return getOpCodeValue() == TR::OverflowCHK; }
+   bool isOverflowCHK()         const { return (getOpCodeValue() == TR::OverflowCHK)         || 
```

Comment:
I am not a huge fan of identity based queries like these. Are we using these two queries anywhere? If not, I would prefer we not create them until they are needed (at which point we can evaluate if they're really the thing that's needed).